### ISSUE TITLE
Using a column value for 'minimum_data' in label_maker.search()

### ISSUE THREAD

**Issue by S-UP** [alteryx/compose#215]
I would like to pass the date when a customer was created as an input for the ```minimum_data``` in ```label_maker.search()```. I understand minimum_data would require a lower bound for the series (per customer, in this case).

The data structure then would go from e.g. Transaction1-NoEvent-NoEvent-Transaction2 to NoEvent (customer creation date)-NoEvent-Transaction1 ...

It seems it is not possible to refer to an existing column that is part of the dataframe used to generate the labels when specifying minimum_data.

Motivation: I don't want to start off with the first transaction on record but the time when we acquired the customer record. In a later stage I would want to add e.g. marketing activities on the customer level. The hypothesis would be that customers who transition more quickly from creating an account to purchasing a product are generally more eager to engage with the brand.

**Comment by jeff-hernandez** [comment]
@S-UP thanks for this feature request! I think this would be a great feature to add. If I understand correctly, we want to use the `min_data` parameter to set the starting point per customer. For example, if we have a series that shows when each customer created their account.

```
customer_id
3    2021-02-03
37   2021-03-14
43   2021-01-02
32   2021-01-03
7    2021-01-14
Name: created_account, dtype: datetime64[ns]
```

Then, we can pass that into the search and get label times starting from the time that the account was created.

```python
lt = lm.search(..., minimum_data=min_data)
```

|   customer_id | time       | made_purchase   |
|--------------:|:-----------|:----------------|
|             3 | 2021-02-03 | True            |
|            37 | 2021-03-14 | True            |
|            43 | 2021-01-02 | True            |
|            32 | 2021-01-03 | True            |
|             7 | 2021-01-14 | True            |

Let me know if this sounds good.

**Comment by S-UP** [comment]
That's what I had in mind. Would be nice to see this addition.

**Comment by jeff-hernandez** [comment]
Nice, thanks for the confirmation! I will keep this issue updated on the progress.

**Comment by S-UP** [comment]
Were you successful in building this feature? Thanks for the update.

**Comment by jeff-hernandez** [comment]
@S-UP this feature is certainly in the roadmap. We are currently working on some major changes in Featuretools. Once that's done, we'll start building this feature. Thanks for following up and sorry for the delay.

**Comment by jeff-hernandez** [comment]
@S-UP this feature is available in the latest release 0.7.0. Sorry for the delay and thanks again for the feature request!
